<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание каталога точек</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>446</width>
     <height>86</height>
    </rect>
   </property>
   <property name="title">
    <string>Открытые таблицы</string>
   </property>
   <widget class="QComboBox" name="cb_tables">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>25</y>
      <width>416</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_unical">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>55</y>
      <width>206</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Только с уникальными полями</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>210</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Файл(файлы) каталога точек</string>
   </property>
   <widget class="QLineEdit" name="ln_pathOut">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>366</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Файл ( файлы ) каталога</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_saveas">
    <property name="geometry">
     <rect>
      <x>397</x>
      <y>20</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Выбрать файл ( директорию)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/save_as_32.png</normaloff>icons/save_as_32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_allInOneFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>90</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Один файл</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_oneObjFile">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>45</y>
      <width>216</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Отдельно для каждого объекта</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>75</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Формат</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_outFormat">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>70</y>
      <width>142</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MapInfo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Excel(xlsx)</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>105</y>
     <width>436</width>
     <height>96</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ключевое поле</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_fields">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>15</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Проекция</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_outCoordSys">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>55</y>
      <width>296</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Выходная проекция</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_coordsys">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Изменить проекцию</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_run">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>315</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>315</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
